src/utils.jl

```diff
@@ -14,3 +14,31 @@ function extract_keywords(ex0)
     end
     return kws, arg
 end
+
+"""
+    wait_for_tracy(;timeout::Float64 = 20.0)
+
+Waits up to `timeout` seconds for `libtracy` to connect to a listening capture
+agent.  If a timeout occurs, throws an `InvalidStateException`.
+"""
+function wait_for_tracy(;timeout::Float64 = 20.0)
```

Comment:
20s feels kind of short.

Reply:
The typical usecase for something like this is:

```
using Tracy, TracyProfiler_jll

# Start the tracy capture agent
p = Tracy.capture("inverter.tracy")

# Wait for tracy to connect
@info("Waiting for Tracy...")
Tracy.wait_for_tracy()

# Run your workload
```

So the amount of time you should be waiting is ~how long it takes the tracy recorder process to startup, by default.  If you need more, you can override `timeout`, of course, but I actually think 20s might be too long; maybe 5s is better, since it should never take longer than that for the connection to be made.

Reply:
Okay, I typically have the tracy UI in a completely different process so for me I set things up so that the workload is ready to go and is just waiting for tracy and then I start up the UI and connect.